core/include/mmcore/MegaMolGraph.h

```diff
@@ -143,6 +144,13 @@ class MEGAMOLCORE_API MegaMolGraph { //: public serializable, public deferrable_
 
     MegaMolGraph_Convenience& Convenience();
 
+    frontend_resources::Command::EffectFunction Parameter_Lambda = [&](const frontend_resources::Command* self) {
+        auto my_p = this->FindParameter(self->parent);
+        if (my_p != nullptr) {
+            my_p->setDirty();
+        }
+    };
```

Comment:
I would be glad if we found a way to somewhow keep such code related to the commands out of the Graph.

Reply:
we can define this lambda anywhere else if you prefer that. even in the commandregistry itself...